src/main/java/ContactsPage.java

```diff
@@ -22,13 +23,21 @@ public class ContactsPage extends BasePage {
     private WebElement contactInformationHeader;
     @FindBy(css = "a.nav-link[routerlink='']")
     private WebElement contactsList;
-    @FindBy(css = "input[formcontrolname='searchInput']\"")
+    @FindBy(xpath = "//*[@id=\"input-search-contact\"]")
     private WebElement searchInputForm;
     @FindBy(css = "b")
     private WebElement foundContact;
     @FindBy(css = "img[src='/assets/icons/trash.svg']")
     private WebElement deleteButton;
 
+    @FindBy(id = "add-contact-modal")
+    private WebElement modalWindowAddContact;
+
+    @FindBy(xpath = "//*[@id=\"contacts-list\"]/div/app-contact-item/div/button[1]")
+    private WebElement fromContactsList;
```

Comment:
I don’t understand what do you mean `fromContactList`, please be more precise in the names.

For example:
`contactCardFields`

Reply:
I fix it